tests/FindAddressTests.html

```diff
@@ -8,63 +8,69 @@
+        <link rel="stylesheet" href="../bower_components/bootstrap/dist/css/bootstrap.css">
+        <link rel="stylesheet" href="../bower_components/esri/css/main.css">
         <style type='text/css'>
-            @import '../../../resources/locate/FindAddress.css';
+            @import '../resources/FindAddress.css';
 
+            body {
+                padding: 25px;
+            }
             #mapDiv {
-                width: 400px;
-                height: 250px;
+                width: 600px;
+                height: 450px;
                 border: 1px solid gray;
             }
+            .find-address {
+                width: 300px;
+            }
         </style>
 
         <!-- JAVASCRIPT -->
-        <script type='text/javascript' src='http://serverapi.arcgisonline.com/jsapi/arcgis/3.4/'> </script>
+        <script type='text/javascript' src='../bower_components/dojo/dojo.js'></script>
         <script type='text/javascript'>
-            var projectUrl = location.pathname.replace(/\/[^\/]+$/, "") + '/';
+            var projectUrl = location.pathname.replace(/\/[^\/]+$/, '') + '/';
 
             require({
                 packages: [{
-                    name: 'agrc',
-                    location: projectUrl + '../../..'
+                    name: 'dart-board',
+                    location: projectUrl + '../'
+                }, {
+                    name: 'esri',
+                    location: projectUrl + '../bower_components/esri'
                 }]
-            },
-            [
-                'agrc/widgets/locate/FindAddress',
-                'dojo/_base/kernel',
+            }, [
+                'dart-board/FindAddress',
+
+                'esri/Map',
+                'esri/views/MapView',
 
-                'dijit/layout/BorderContainer',
-                'dijit/layout/ContentPane',
                 'dojo/domReady!'
-            ], 
-            function(
-                find, 
-                kernal
-                ) {
-                kernal.global.base = new esri.layers.ArcGISTiledMapServiceLayer("http://mapserv.utah.gov/arcgis/rest/services/BaseMaps/Vector/MapServer", 
-                    {
-                        id: "vector",
-                        visible: true
-                    });
+            ], function (
+                FindAddress,
 
-                var map = new esri.Map('mapDiv');
-                map.addLayer(base);
+                Map,
+                MapView
+                ) {
+                var map = new Map({basemap: 'streets'});
+                var mapView = new MapView({
+                    map,
+                    container: 'mapDiv'
+                });
 
-                var geocode = new find({ 
-                    map: map, 
-                    apiKey: 'AGRC-ApiExplorer'
+                new FindAddress({
+                    mapView,
+                    apiKey: 'AGRC-63E1FF17767822',
```

Comment:
I forgot to remove it. I would love to have a key that we can put on test pages that works with localhost. Do you want to create a generic one that's not connected to my account?